docs/DeveloperGuide.md

```diff
@@ -257,13 +257,14 @@ _{Explain here how the data archiving feature will be implemented}_
 
 **Target user profile**:
 
-* has a need to manage a significant number of contacts
+* university student who is looking for internships / full time jobs
+* has a need to manage a significant number of companies, their events, and their contact people during job seeking
 * prefer desktop apps over other types
 * can type fast
 * prefers typing to mouse interactions
 * is reasonably comfortable using CLI apps
 
-**Value proposition**: manage contacts faster than a typical mouse/GUI driven app
+**Value proposition**: manage companies, their events, and their contact people faster than a typical mouse/GUI driven app
 
```

Comment:
I think for the value proposition, you can add another explanation where InternBuddy tries to combine managing contacts and managing Events in one application, which help users to less juggle around between different applications.

Reply:
I think the statement in the values proposition already implies this, so it's unnecessary to add another statement